### ISSUE TITLE
Dvorak find links are unreadable

### ISSUE THREAD

**Issue by pickfire** [philc/vimium#4027]
**Describe the bug**
When using find links with dvorak keys, the keys are unreadable, especially `,` vs `.` vs `'` given that the text shown is so small.

**To Reproduce**

1. Set config "characters used for link hints" to `aoeu',.p;qjk`
1. Press `f`
2. See the screenshot
![image](https://user-images.githubusercontent.com/4687791/158778224-114f3c1d-2978-4438-af51-43438a1ccd45.png)

Include a screenshot if applicable.

**Browser and Vimium version** 98.0.1  arch linux Linux 5.16.14-zen1-1-zen #1 ZEN SMP PREEMPT Fri, 11 Mar 2022 17:40:33 +0000 and 1.67.1

**Comment by gdh1995** [comment]
Um `markerMatcher` of `LinkHints` doesn't follow `mapKey` in the `Custom key mappings` option, so you may need to change fonts using CSS:
![image](https://user-images.githubusercontent.com/5547703/158932948-b732139c-f025-44c3-bbc5-7abf25500d6c.png)

Or you may try my customized version of Vimium, named Vimium C (https://github.com/gdh1995/vimium-c), and it supports a more powerful `mapKey` and `LinkHints` also uses it. The doc is https://github.com/gdh1995/vimium-c/wiki/Use-in-another-keyboard-layout

**Comment by mcp292** [comment]
Try these as your link hint characters instead: `oeu.nthc`. I've been using these on Dvorak for years. They're the three inner home row resting positions, and the two above.

**Comment by pickfire** [comment]
> Try these as your link hint characters instead: oeu.pnthcg. I've been using these on Dvorak for years. They're the three inner home row resting positions of each hand, and the two above.

I think it might be better for it to be within a hand since my right hand might move to the mouse sometimes, at least left hand is always at the keyboard.

But I think this should be fixed by styles or something, the hardest to read is `,` and `.` for me, I need to look closely to identify them, maybe we can make the link larger or choose a default font that can easily distinguish them?

**Comment by mcp292** [comment]
> I think it might be better for it to be within a hand since my right hand might move to the mouse sometimes, at least left hand is always at the keyboard.

Isn't the point of hints and Vimium in general to not use the mouse?

**Comment by pickfire** [comment]
Yes, that is true, but based on the default keys, the hint forwarding can be done with one hand.

**Comment by mcp292** [comment]
Another solution is to use what you have, minus either the comma or period; use either, not both.

> Yes, that is true, but based on the default keys, the hint forwarding can be done with one hand.

Aren't the defaults for QWERTY, and thus, not designed for single handed use.

**Comment by pickfire** [comment]
Yes, but after swapping to dvorak, if I remove either the comma and period it seemed weird, suddenly either a third or fourth upper finger don't get to press anything, but pinky still need to press `'`.
